doc/examples/histology_resdnn.py

```diff
@@ -107,7 +107,7 @@
 of the corpus callosum and a single-fiber population should be visible.
 """
 
-camera = {}
+camera = dict({})
 camera['zoom_factor'] = 0.85
 camera['view_position'] = np.array([(ori_shape[0] - 1) / 2.0,
```

Comment:
you can initialize it like you did above (meaning, ` camera = {'zoom_factor': 0.85 ....}`)

Reply:
Done in 5517829.